src/app.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Form from './form.js'
+import {appState} from './app-state.js'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = appState
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+  handleSubmit(event) {
```

Comment:
Define `handleSubmit` in your form component. The method in your `App` class should be more generic, like `addFlashCard` that takes a `flashCard` and adds it to `state`.